src/workerd/util/autogate.h

```diff
@@ -42,9 +40,6 @@ class Autogate {
   static void initAutogate(
       capnp::List<capnp::Text>::Reader autogates);
 
-  // Convenience method for tests to use to invoke initAutogate()
-  static void initEmptyAutogateForTesting();
```

Comment:
Might be a good idea to keep this for future use, but not-blocking.

Reply:
Yeah, I wasn't sure how much of the autogate-specific changes to leave in.

For this function in particular, though, it occurred to me after the fact that it might be more broadly useful if it were implemented as something taking a list of autogate name strings.  I can try adding that in a separate PR.